03-11-2018/HybernateExample/src/main/java/com/hibernate/service/HibernateDatabaseProcessing.java

```diff
@@ -0,0 +1,12 @@
+package com.hibernate.service;
+
+import com.hibernate.daoImpl.DatabaseConnectionDaoImpl;
+
+public class HibernateDatabaseProcessing 
+{
+	public static void main(String args[]) throws ClassNotFoundException
+	{
```

Comment:
why main method created in service class. Main method is not acceptable at service and dao. Dont do this again